aptos-move/aptos-aggregator/src/tests/identifier_mappings.rs

```diff
@@ -1,92 +1,92 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::types::{DelayedFieldID, DelayedFieldValue, TryFromMoveValue, TryIntoMoveValue};
-use aptos_types::delayed_fields::bytes_and_width_to_derived_string_struct;
-use claims::{assert_err, assert_ok};
-use move_core_types::value::{
-    IdentifierMappingKind,
-    MoveStructLayout::Runtime,
-    MoveTypeLayout,
-    MoveTypeLayout::{Address, Bool, Struct, Vector, U128, U64, U8},
-};
-use move_vm_types::values::Value;
-use once_cell::sync::Lazy;
-use test_case::test_case;
-use DelayedFieldValue as A;
-use IdentifierMappingKind as K;
+// use crate::types::{DelayedFieldID, DelayedFieldValue, TryFromMoveValue, TryIntoMoveValue};
```

Comment:
What happened to these tests? Are all these tests no longer relevant?

Reply:
Commented out while coding, returned the once which do `DelayedFieldValue <--> Value` conversions as these are still relevant. For `DelayedFieldID`, not really, because now we do `SizedID <--> bytes` inside Move. I added a TODO there while I have not added these tests

Reply:
All tests returned